consensus/validator.go

```diff
@@ -293,6 +294,13 @@ func (consensus *Consensus) onCommitted(msg *msg_pb.Message) {
 		}
 	}
 
+	ConsensusCounterVec.With(prometheus.Labels{"consensus": "bingo"}).Inc()
+	numSig := float64(consensus.NumSignaturesIncludedInBlock(blockObj))
```

Comment:
can we add this counter at BINGO? It's a relatively expensive operation so we only want to do it once.

Reply:
BINGO message is in node module, can't use the variables defined in consensus module.

Reply:
let me add a func then.

Reply:
done